Files/UserControls/LayoutModes/GridView.xaml.cs

```diff
@@ -11,9 +11,9 @@
 
 namespace Files
 {
-    public sealed partial class PhotoAlbum : BaseLayout
+    public sealed partial class GridView : BaseLayout
```

Comment:
This should be GridViewBrowser.

Reply:
Thanks for picking this up, it should be fixed now